src/libraries/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.Unescaping.cs

```diff
@@ -65,6 +65,33 @@ public static string GetUnescapedString(ReadOnlySpan<byte> utf8Source, int idx)
             return utf8String;
         }
 
+        // TODO: Borrowing this from https://github.com/dotnet/runtime/pull/32669/files#diff-82b934371fff193b37c85635239bb6ebR70
+        // Remove when that PR is in or remove from #32669 if this gets in first.
+        public static ReadOnlySpan<byte> GetUnescapedSpan(ReadOnlySpan<byte> utf8Source, int idx)
+        {
+            // The escaped name is always >= than the unescaped, so it is safe to use escaped name for the buffer length.
+            int length = utf8Source.Length;
+            byte[]? pooledName = null;
+
+            Span<byte> utf8Unescaped = length <= JsonConstants.StackallocThreshold ?
```

Comment:
nit - put the condition of the conditional operator inside parenthesis:
```suggestion
            Span<byte> utf8Unescaped = (length <= JsonConstants.StackallocThreshold) ?
```

Reply:
There is a whole bunch of ternary operator usage in the code base. Most of what I see don't use brackets around the condition (especially when it is a simple, single condition).

Search for ` ? ` usages in code.

https://github.com/dotnet/runtime/blob/e631031c93175bf5a1ec01aa6a8cf8330f168153/src/libraries/Common/src/System/Security/Cryptography/RSAOpenSsl.cs#L408
https://github.com/dotnet/runtime/blob/e631031c93175bf5a1ec01aa6a8cf8330f168153/src/libraries/Common/src/System/Text/DBCSDecoder.cs#L55
https://github.com/dotnet/runtime/blob/e631031c93175bf5a1ec01aa6a8cf8330f168153/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L236
https://github.com/dotnet/runtime/blob/e631031c93175bf5a1ec01aa6a8cf8330f168153/src/libraries/System.Private.CoreLib/src/System/String.cs#L370
https://github.com/dotnet/runtime/blob/e631031c93175bf5a1ec01aa6a8cf8330f168153/src/libraries/System.Private.CoreLib/src/System/Collections/ObjectModel/ReadOnlyCollection.cs#L94

Complex conditions tend to have brackets (but even that seems to be based on personal preference):
https://github.com/dotnet/runtime/blob/e631031c93175bf5a1ec01aa6a8cf8330f168153/src/libraries/System.Private.CoreLib/src/System/String.cs#L466
https://github.com/dotnet/runtime/blob/e631031c93175bf5a1ec01aa6a8cf8330f168153/src/libraries/System.Web.HttpUtility/src/System/Web/HttpUtility.cs#L100